### PR TITLE
Bug/vgnwb 368

### PR DESCRIPTION
**Pull request by sudheerad9**
# Guidelines for Pull Requests

If you haven't yet read our code review guidelines, please do so, You can find them [here](https://diging.atlassian.net/wiki/spaces/DIGING/pages/2256076801/Code+Review+Guidelines).

Please confirm the following by adding an x for each item (turn `[ ]` into `[x]`).

- [x] I have removed all code style changes that are not necessary (e.g. changing blanks across the whole file that don’t need to be changed, adding empty lines in parts other than your own code)
- [x] I am not making any changes to files that don’t have any effect (e.g. imports added that don’t need to be added)
- [x] I do not have any sysout statements in my code or commented out code that isn’t needed anymore
- [x] I am not reformatting any files in the wrong format or without cause. 
- [x] I am not changing file encoding or line endings to something else than UTF-8, LF
- [x] My pull request does not show an insane amount of files being changed although my ticket only requires a few files being changed
- [x] I have added Javadoc/documentation where appropriate
- [x] I have added test cases where appropriate
- [x] I have explained any part of my code/implementation decisions that is not be self-explanatory

## Please provide a brief description of your ticket 
(you can copy the ticket if it hasn't changed) 

fix signup error
  
## Anything else the reviewer needs to know?

... describe here ...

### CODE REVIEW

File: annotations/views/user_views.py [hunk 1]
```diff
@@ -227,3 +229,22 @@ def get_weekly_annotations(self, user):
 				'count': weekly_annotations.get(week.datetime, 0)
 			})
 		return result
+	
+
```
Comment (tjquinn1) [inline annotations/views/user_views.py]: remove extra line

File: annotations/views/user_views.py [hunk 2]
```diff
@@ -227,3 +229,22 @@ def get_weekly_annotations(self, user):
...
+@api_view(["POST"])
+@authentication_classes([])
```
Comment (tjquinn1) [inline annotations/views/user_views.py]: is this necessary if you are using no classes?

File: annotations/views/user_views.py [hunk 3]
```diff
@@ -227,3 +229,22 @@ def get_weekly_annotations(self, user):
...
+@permission_classes([])
```
Comment (tjquinn1) [inline annotations/views/user_views.py]: same as above

File: annotations/views/user_views.py [hunk 4]
```diff
@@ -227,3 +229,22 @@ def get_weekly_annotations(self, user):
...
+def create(request):
+	try:
+		user = VogonUser.objects.create(**request.data)
+		user.save()
+		refresh = RefreshToken.for_user(user)
+		return Response({
```
Comment (tjquinn1) [inline annotations/views/user_views.py:239]: This should also return a response status 201 if the created succeeded

**Review by tjquinn1 — CHANGES_REQUESTED** [review]

File: annotations/views/user_views.py [hunk 5]
```diff
@@ -227,3 +229,19 @@ def get_weekly_annotations(self, user):
 				'count': weekly_annotations.get(week.datetime, 0)
 			})
 		return result
+	
+@api_view(["POST"])
+def create(request):
```
Comment (tjquinn1) [inline annotations/views/user_views.py:234]: Is this necessary? Whats wrong with /users?

Comment (sudheerad9) [reply annotations/views/user_views.py:234]: Seperating user creation and signup. In user creation we need not return the access and refresh tokens, for signup UI is setting those in local storage. So the signup API creates user and then returns the refresh token for users.

**Review by jophals — APPROVED** [review]
Approved for merge.

**Pull request merged**